book/website/community-handbook/local-build.md

```diff
@@ -0,0 +1,67 @@
+(ch-local-build)=
+# How to build Jupyter Book locally
+
+## But why build locally?
+It's always handy to be able to preview any changes you have been working on as you go - you can be confident that changes you have made are accurate and as intended. 
+A nice way to do this is to use the underlying Jupyter Book tool to build the book locally.
+
+This is useful because it allows you to preview any changes you have made on your local machine *before* you push your changes to a remote branch. 
+You can then decide if you are happy with the result and push your changes to the remote branch thus helping to keep Pull Request conversations and commit histories a bit cleaner.
+
+## Step-by-step guide
+1. Install miniconda https://conda.io/projects/conda/en/latest/user-guide/install/index.html#regular-installation
+2. Run `conda init` in your terminal. You should see `(base)` in your prompt indicating that conda was successfully installed and you are now in it's base environment.
```

Comment:
We maybe want to be explicit about "open your terminal app and run the `conda init` command"

Reply:
Yes, i agree with this.
And to add to it,  my cmd did not recognize the `conda init` , i had to use the Anaconda prompt(miniconda) terminal  to run it.

Reply:
Anaconda and miniconda are different installations. Are you sure you installed miniconda, as per the instructions? Did your machine already have anaconda installed before you started?

Reply:
Yes, i followed the instruction for installation https://conda.io/projects/conda/en/latest/user-guide/install/index.html#regular-installation, I selected for windows.
No i never had the Anaconda on my machine.

Reply:
Huh, ok. This is probably a "things are different on Windows" scenario, so we should have a nested bullet point here saying "on windows, do this instead"

Reply:
Hi both - hopefully I've addressed the comments in this conversation in commit c3c7dda3d70655f20d8429d736b9cc7fa112f120 . Namely adding more narrative and the point about needing Anaconda prompt on Windows

Reply:
Are you sure you pushed the narrative changes? I'm examining commit [`c3c7dda` (#2922)](https://github.com/alan-turing-institute/the-turing-way/pull/2922/commits/c3c7dda3d70655f20d8429d736b9cc7fa112f120) but it looks pretty similar to me? I'm imagining that we get rid of the bullet points under "Why did we recommend using (mini)conda?"

Reply:
ah I was looking at the wrong section - I added some very light touch edits on the build instructions in c3c7dda. Ok no problem I'll have a crack at the miniconda section.

Reply:
2ca45fa43a1a2a2597b8387e641b2f6ab573334a is hopefully more what you had in mind, and the title is now in line with the toc